chalk-parse/src/parser.lalrpop

```diff
@@ -58,8 +58,11 @@ WellKnownTrait: WellKnownTrait = {
      "#" "[" "lang" "(" "unsize" ")" "]" => WellKnownTrait::Unsize,
 };
 
+StructReprC: () = "#" "[" "repr(C)" "]";
+StructReprPacked: () = "#" "[" "repr(packed)" "]";
```

Comment:
I think for this we should just parse any identifier and then generate a list of identifiers in the AST, perhaps, which lowering can match against "C" or "packed", or else have a `match` statement that examines the identifier in the parser:

```
StructRepr: XXX = "#" "[" "repr" "(" <name:Ident>")" "]" => {
    match name {
        "C" => ...
        "repr" => ...
    }
}
```

Reply:
I implemented your suggestion so that StructRepr now parses to an Atom.